### ISSUE TITLE
Compile instructions for windows

### ISSUE THREAD

**Issue by alfie-nsugh** [EQMG/Acid#98]
Could you describe (in a bit more detail) instructions on how to compile this project on windows, I tried with visual studio and mingw-w64, my personal preference is for the latter but any of these two will do, I got it to work on linux but I couldn't get it to work on windows.

**Describe the solution you'd like**
For msvc or mingw-w64, to have clear cut instructions on how to compile the project.

**Comment by mattparks** [comment]
I've used MinGW x64 from CLion with success in the past, I have Windows build documentation here: https://github.com/EQMG/Acid/wiki/Setup-Windows
You will need the Vulkan SDK and OpenAL SDK installed, if you don't the CMake setup will fail.

The master branch built just fine on my Visual Studio 2019 IDE, if you have further issues can you post your IDE version, and potentatily more specific errors?

**Comment by alfie-nsugh** [comment]
I will first try it on another windows machine first, maybe I cluttered it a little with msys2. Thank you for your swift response

**Comment by alfie-nsugh** [comment]
I tried it on another pc and got the same problem:
![afbeelding](https://user-images.githubusercontent.com/40834779/89098026-5200a680-d3e4-11ea-93b2-d04009cf35b8.png)
So basically a lot of compiler errors, I don't know whether they're readable or helpful, to me it's quite opaque, if they're of use I can post more screenshots. 
![afbeelding](https://user-images.githubusercontent.com/40834779/89098980-6d6faf80-d3ec-11ea-8772-3df7ddd21413.png)
This is my cmake-gui config, I tried with mingw-w64 64 bit, the posix version as well as the win32 version. In both cases the cmake configures and generates properly, but while making the same error occurs. Maybe gnu 8.1.0 is outdated, since on ubuntu(ubuntudde) 20 it uses 9.3.0 and it compiles and runs successfully there.
 I'll try compiling it with clion, maybe it'll work there.
